forestci/forestci.py

```diff
@@ -291,10 +325,11 @@ def random_forest_error(
         return V_IJ_unbiased
 
     if V_IJ_unbiased.shape[0] <= 20:
-        print("No calibration with n_samples <= 20")
+        print("No calibration with n_samples <= 20: consider using more n_estimators in your model, for more accurate ci and to avoid negative values.")
```

Comment:
For PEP8's sake:

```suggestion
        print("No calibration with n_samples <= 20: ", 
                 "consider using more n_estimators in your model, ", 
                 "for more accurate ci and to avoid negative values.")
```